added tech journey (#23)
iamshadmirza/TheNextBigWriter: Tech Journey/How I got into tech.md

## How I got into tech?

> To begin with my writing journey, I feel that I must introduce myself, and what's better than to write about my journey in tech and give an insight into what brought me here.

Reviewer: ```suggestion
To begin with my writing journey, I felt that I shall start with introducing myself, and what's better than to write about my journey in tech and give an insight into what brought me here.
```

#### The Beginning

> I have been enthusiastic about how the software stuff works, what goes behind the screen of my computer and I always imagined how they're made. I have been keenly interested in computer science since school whether they taught us QBasic, LOGO, VisualC++, SQL, HTML, programming in C++ or networking. Anything related to computer science lured me to read and know more about it. What added to my childhood fantasies was the movie - Jobs(2013) which got me really interested in the life of the tech giant and visionary Steve Jobs. Thereafter, I read several books on his life and wished to be as great as him someday. But as he once said- 

Reviewer: ```suggestion
I have been enthusiastic about how the software stuff works, what goes behind the screen of my computer and I always imagined how they're made. I have been keen in computer science since school whether they taught us QBasic, LOGO, VisualC++, SQL, HTML, programming in C++ or networking. Anything related to computer science lured me to read and know more about it. What added to my childhood fantasies was the movie - Jobs(2013) which got me really interested in the life of the tech giant and visionary Steve Jobs. Thereafter, I read several books on his life and wished to be as great as him someday. But as he once said- 
```

>"When you grow up, you tend to get told the world is the way that it is, and your life is just to live your life inside the world and try not to bash into the walls too much. But that's a very limited life."

> I always wanted to study Computer Science but getting admission in a good college with CS/IT stream is no more an easy deal. You got to almost kill yourself studying Physics, Chemistry and Maths to pass those entrance exams with good marks. I passed the entrance exams I gave but was just on the borderline with the cut-off marks in hand, which led to me a third-tier college affiliated to AKTU, although, I was happy that it was quite famous for its Computer Science Graduates doing really well with decent placement offers in big firms. Meanwhile, I started writing articles for [fossbytes.com](https://fossbytes.com/author/pragati/).

Reviewer: ```suggestion
I always wanted to study Computer Science but getting admission in a good college with CS/IT stream is no more an easy deal. You almost kill yourself studying Physics, Chemistry and Maths to pass those entrance exams with good marks. I passed the entrance exams I gave but was just on the borderline with the cut-off marks in hand, which led to me a third-tier college affiliated to AKTU, although, I was happy that it was quite famous for its Computer Science Graduates doing really well with decent placement offers in big firms. Meanwhile, I started writing articles for [fossbytes.com](https://fossbytes.com/author/pragati/).
```

#### Outset of College

> As I bumped into college life, I got introduced to the cool and awesome technical student societies/clubs of the college, and then it become my prime goal to get in the best of them and try to improve my technical skills and get the best out of what I have got. Moreover, I kept reading books about Elon Musk, Linus Torvalds and Bill Gates. In the first year itself, I was introduced to the world of competitive coding, I thought maybe I can do this. So, I began practicing on platforms like [CodeChef](https://www.codechef.com/users/pragativerma18) and [HackerRank](https://www.hackerrank.com/itispragativerma) etc. It went smooth for sometime, I was able to do 2-3 questions in a long challenge but there came a time I realised that this wasn't I was looking for. I began exploring more options, where I thought that web development might be fun, so I switched to development. Competitive Coding is important due to its own reasons but I wanted to create products that may or may not be impactful but can be at least useful.

Reviewer: ```suggestion
As I bumped into college life, I got introduced to the cool and awesome technical student societies/clubs of the college, and then it became my prime goal to get in the best of them and try to improve my technical skills and get the best out of it. Moreover, I kept reading books about Elon Musk, Linus Torvalds and Bill Gates. In the first year itself, I was introduced to the world of competitive coding, I thought maybe I can do this. So, I began practicing on platforms like [CodeChef](https://www.codechef.com/users/pragativerma18) and [HackerRank](https://www.hackerrank.com/itispragativerma) etc. It went smooth for sometime, I was able to do 2-3 questions in a long challenge but there came a time when I realised that this isn't something I was looking for. I began exploring more options, where I thought that web development might be fun, so I switched to development. Competitive Coding is important due to its own reasons but I wanted to create products that might be impactful and can be useful.
```

#### Confusion and Rejections
However, no story in life goes that easy. I got rejected from recruitment of all the technical societies in my college. These rejections got me to a halt, I thought maybe I am not competent enough for this technical world and I should rather focus on academics, but my friends motivated me through this ugly phase and introduced me to the '100 Days Of Code Challenge'. This challenge introduced me to many good developers across the country. I began the challenge and decided to work on my frontend development skills. I learnt HTML,CSS and Bootstrap during the second semester.

#### Internship and Hackathons

> Meanwhile, I was exploring the world of open source software and decided to learn how to use git and GitHub, such that I can try to contribute to open source in whichever way possible. I also kept looking for internship opportunities and fortunately, I got a web development intern role at a startup named [Uneako](http://uneako.com/). We also started exploring hackathons nearby our college, our first hackathon being- hackit@BVP1.0 at BVP,Delhi. It was an enriching experience to be part of these hackathons and meetups, as they open a window to the better in you. They give us a chance to introspect what we have been lacking or what more is to be done to be the winner in your chosen arena of work or ambition. Often, they are the inspiration of new beginnings or the genesis of better future.

Reviewer: ```suggestion
Meanwhile, I was exploring the world of open source software and decided to learn how to use git and GitHub, such that I can try to contribute to open source in whichever way possible. I also kept looking for internship opportunities and fortunately, I got a web development intern role at a startup named [Uneako](http://uneako.com/). We also started exploring hackathons nearby our college, our first hackathon being hackit@BVP1.0 at BVP, Delhi. It was an enriching experience to be part of such hackathons and meetups, as they open a window and help you evolve to be a better person. They give us a chance to introspect what we have been lacking or what more is to be done to be the winner in your chosen arena of work or ambition. Often, they are the inspiration of new beginnings or the genesis of better future.
```

#### Open Source and Hacktoberfest
Open Source is a fascinating world in itself. It relies on contribution from each and every individual to bring forth an innovation for tomorrow, while sharing every resource in utter transparency. I was trying to make contributions to open source organizations but nothing much was turning up for me. I started learning Python and Web Development in Django to know more about the backend of a website.

Then came Hacktoberfest, the celebration season for open source, but it was disheartening to notice that most of my juniors and colleagues were unaware of this month-long contest. So, I, along with my friends decided to make a [website](https://hacktoberfest.netlify.com/) and a [repository](https://github.com/Open-Source-Contributors-JSS/Hacktoberfest2019) to promote hacktoberfest and to help everyone learn about the usage of git and GitHub. To our surprise, we received an overwhelming response from students of our college as well as from people across the globe, our repository had more than 300 stars and more than 800 forks.

#### Google Code-In and Thereafter

> Then, I came across [Google Code-In](https://codein.withgoogle.com/), I thought if I could have known and participated in it when I was in school, it would've been great. However, I saw a chance that I could apply as a mentor, so, I did. I was selected as a Google Code-In Mentor for [CircuitVerse.org](https://circuitverse.org/) and it was truly an incredible journey to see so many young individuals participate and contribute to open source. Thereafter, I mentored a few other open source events like- NWoC(IIT Patna), KWoC KOSS(IIT Kharagpur), JMOC(JIIT Noida) and currently I am [GSSoC](https://www.gssoc.tech/)(GirlScript Foundation) Mentor for 'Water Monitoring System'.

Reviewer: ```suggestion
Then, I came across [Google Code-In](https://codein.withgoogle.com/), I thought if I could have known and participated in it when I was in school, it would've been great. However, I saw a chance that I could apply as a mentor, so I did. I was selected as a Google Code-In Mentor for [CircuitVerse.org](https://circuitverse.org/) and it was truly an incredible journey to see so many young individuals participate and contribute to open source. Thereafter, I mentored a few other open source events like- NWoC(IIT Patna), KWoC KOSS(IIT Kharagpur), JMOC(JIIT Noida) and currently I am [GSSoC](https://www.gssoc.tech/)(GirlScript Foundation) Mentor for 'Water Monitoring System'.
```

> This journey of mine may not be a big inspiration story, however, this is what made me what I am today.

Reviewer: ```suggestion
My journey may not sound as a big inspiration story, however, this is what made me what I am today.
```

This helped me learn what Steve Jobs said:

>"Life can be much broader once you discover one simple fact. And that is that everything around you that you call life, was made up by people that are no smarter than you. And you can change it. You can influence it. You can build your own things that other people can use."

#### What am I today?

> Today, I am an aspiring full stack web developer, learning to make awesome websites that can cater user demands in an efficient and better way. Furthermore, my team is also a [SIH](http://sih.gov.in/) finalist this year. Also, I am eager to share my knowledge and understanding of things, so, I decide to write, and hence, I am here with the Next Big Writer.

Reviewer: ```suggestion
Today, I am an aspiring full stack web developer, learning to make awesome websites that can cater user demands in an efficient and better way. Furthermore, my team is also a [SIH](http://sih.gov.in/) finalist this year. Also, I am eager to share my knowledge and understanding of things, so I decided to write, and hence, I am here with the Next Big Writer. 
```
